CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+Please follow these beautiful coding standards set by Airbnb when writing code for inclusion in this repository.
+<https://github.com/airbnb/javascript/tree/master/react>
+
+Note we have a code of conduct (code of conflict), please follow it in all your interactions with the project.
```

Comment:
There has been an update with our contributing guidelines.

Please update it and manage it accordingly:
https://github.com/snek-at/front/blob/master/CONTRIBUTING.md